modules/integration_aws-backup/conf/00-aws-backup-failed.yaml

```diff
@@ -0,0 +1,22 @@
+module: AWS Backup
+name: failed
+id: backup_failed
+
+transformation: ".min(over='1h')"
+aggregation: true
+filtering: "filter('namespace', 'AWS/Backup')"
+value_unit: "count"
```

Comment:
this seems not really useful or relevant compared to not define unit at all

Reply:
Addressed: https://github.com/claranet/terraform-signalfx-detectors/pull/457/files#diff-0eec7b99d6548a77ff0d378c3db0de6cae5c0527a186aeab8ed197d0e80904a3R7